package.json

```diff
@@ -135,6 +135,7 @@
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-typescript": "^0.14.0",
+    "focus-trap": "^5.0.0",
```

Comment:
@moog16 I'm requiring this here to prevent sub-packages from installing it twice. I'm updating the version to the `5.x` branch because [dependencies have updated to use it](https://github.com/material-components/material-components-web/blob/master/packages/mdc-dialog/package.json#L31) and conflicts result if not.

Reply:
Let's remove this. The repository does not depend on focus-trap.

Reply:
It looks used: https://github.com/material-components/material-components-web-react/search?utf8=%E2%9C%93&q=focus-trap&type=

Reply:
ah you're right - in the tests. Thanks!